src/pages/index.js

```diff
@@ -249,21 +249,20 @@ const editFormValidator = new FormValidator(settings, profileFormElement);
 editFormValidator.enableValidation();
 const addCardFormValidator = new FormValidator(settings, cardFormElement);
 addCardFormValidator.enableValidation();
-
-/* const deleteCardVerifyValidator = new FormValidator(
+const deleteCardVerifyValidator = new FormValidator(
   settings,
   submitDeleteCardContainer
 );
 deleteCardVerifyValidator.enableValidation();
 const avatarImageFormValidator = new FormValidator(settings, avatarFormElement);
 avatarImageFormValidator.enableValidation();
- */
+
 /* -------------------------------------------------------------------------- */
 /*                                  API Stuff                                 */
 /* -------------------------------------------------------------------------- */
 
 /* Fetching and rendering card data from server */
-/* api.getInitialCards().then((cardData) => {
+api.getInitialCards().then((cardData) => {
```

Comment:
You need to combine the requests of the user's info and the cards together in `Promise.all` because it might happen so that the `_id` has not been received yet but the cards have and your cards will be rendered not correctly. 
More info is here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all

```js

Promise.all([getUserInfo(), getCards()])
// destructure the response
  .then(([userData, cards]) => {
      // set all the data
  })
  .catch(err => {
    // catch possible errors
  });

```